index.ts

```diff
@@ -0,0 +1,42 @@
+module browser_fs {
+    export class SourceFile {
```

Comment:
Add JSDoc comments to public classes and methods (only if they are not self-explanatory)